EIPS/eip-4938.md

```diff
@@ -0,0 +1,57 @@
+---
+eip: 4938
+title: "eth/67: Removal of GetNodeData"
+author: Marius van der Wijden (@MariusVanDerWijden), Felix Lange <fjl@ethereum.org>, Gary Rong <garyrong@ethereum.org>
+discussions-to: https://github.com/ethereum/devp2p/issues/202
```

Comment:
Must be an Ethereum Magicians thread.

Reply:
This I don't understand. We have always used the devp2p issue tracker for the discussions of p2p protocol EIPs. Why does it have to change all of a sudden?

Reply:
What I mean is, `discussions-to` of EIP-868, EIP-778, EIP-1459 was allowed to be a link to an ethereum/devp2p issue. For past eth protocol updates, `discussions-to` was a link to an ethereum/EIPs issue.

EIP-1 says an Ethereum Magicians thread is 'preferred'. It doesn't say 'mandatory'.